buildSrc/src/main/groovy/io/micronaut/gradle/CopyAssetsTask.groovy

```diff
@@ -35,16 +35,22 @@ class CopyAssetsTask extends DefaultTask {
         new File(output.get().absolutePath + "/" + RenderSiteTask.DIST)
     }
 
+    List<String> recursiveIncludes(String[] extensions) {
+        extensions.collect{"*"+it} + extensions.collect {"**/*"+it}
```

Comment:
`extensions.collect{"*"+it} + extensions.collect {"**/*"+it}`

Probably : 

`extensions.collect { ["*$it", "**/*$it"] }.flatten() as List<String>` is clearer